modules/dists/BlockDist.chpl

```diff
@@ -1568,6 +1613,60 @@ proc BlockArr.doiCanBulkTransferStride() param {
   return useBulkTransferDist;
 }
 
+proc BlockArr.oneDData {
```

Comment:
What do you think of adding a quick check against ``defRectSimpleDData`` here, in case someone calls this function without check that param first?